README.md

```diff
@@ -2,10 +2,14 @@
 
 This repo hosts [Synthea](https://github.com/synthetichealth/synthea)-generated
 sample [FHIR](https://www.hl7.org/fhir/) bulk export results,
-useful for testing downstream workflows.
+useful for testing downstream workflows. It also hosts a script for generating samples of
+custom sizes. See the [regenerating a dataset section below](#regenerating-a-dataset) for
+instructions on generating custom sized datasets.
 
 ## Downloads
 
+We've created several pre-zipped bundles of sample patients if you only
```

Comment:
Sentence cuts off

Reply:
Just going to remove this line. It's vestigial from before I made edits to the introduction above.